### ISSUE TITLE
Refined Storage wireless range increase upgrades don't work properly

### ISSUE THREAD

**Issue by Sethy152** [EnigmaticaModpacks/Enigmatica8#362]
### Modpack Version

2.5.1

### Describe your issue.

The range increase upgrades for the wireless transmitter from Refined Storage don't work as intended. The tooltip in the advancements tree says something like "each upgrade incrementally increases the range, but also the power consumption". However, in practice it only increases the range by 8 blocks. With only 4 upgrade slots (and how expensive they are) it must be a bug.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

### Discord Username

The Magma Psychic#5377

**Comment by Kanzaji** [comment]
That is RefinedStorage default behaviour >.> However this should be changed as it is really really low, making everyone pretty much just rush for the infinite card.
No idea about the tooltip however. Will have to check that.

**Comment by Sethy152** [comment]
There isn't an infinite card in E8. At least, not by that name. For now I just edited the config to give each upgrade 500 blocks extra range.

**Comment by Kanzaji** [comment]
Yup, my bad. It was actual block this time, but there is an option for infinite range for RS in E8 :D

**Comment by Sethy152** [comment]
Where is it? I've looked through a few times, and there's stuff for other storage mods but I didn't see any for RS. There's the receiver and transmitter blocks, but that is more of an extra beacon than global range.

**Comment by NielsPilgaard** [comment]
I've increased the range quite a bit in-dev :)
